google-ads-examples/src/main/java/com/google/ads/googleads/examples/errorhandling/HandleRateExceededError.java

```diff
@@ -207,11 +207,15 @@ public void run() {
               MutateAdGroupCriteriaResponse response =
                   adGroupCriterionServiceClient.mutateAdGroupCriteria(
                       Long.toString(customerId), operations, false, true);
-
-              System.out.printf("Added %d ad group criteria:%n", response.getResultsCount());
-              for (MutateAdGroupCriterionResult result : response.getResultsList()) {
-                System.out.println(result.getResourceName());
-              }
+              System.out.printf("%d operations validated.%n", operations.size());
+              // The result set will be empty while validateOnly is set to true in the
```

Comment:
These comments aren't easy to understand since we are not actually creating a MutateAdGroupCriteriaRequest, but instead are using the flattened method.
Since a key point of this example is the validateOnly parameter, I suggest creating a MutateAdGroupCriteriaRequest so readers can clearly see the call to the builder method for `validateOnly`.

Reply:
If we are trying to focus on demonstrating validate only, perhaps this commented out section should just be removed entirely? If we decide to keep it, perhaps, we can stick an 'if' block around it to execute or not based on if validate only is set or not.

Reply:
I'm okay with removing the comment.

Reply:
How about:
1. Switch to constructing a `MutateAdgroupCriteriaRequest` as mentioned above.
2. Making this comment simply:

> The result set will be empty since validateOnly is set to true in the MutateAdGroupCriteria request.

Reply:
+1 I like that idea.

Reply:
See new commit.